packages/mui-material/src/CardContent/CardContent.js

```diff
@@ -5,6 +5,9 @@ import { unstable_composeClasses as composeClasses } from '@mui/base';
 import styled from '../styles/styled';
 import useThemeProps from '../styles/useThemeProps';
 import { getCardContentUtilityClass } from './cardContentClasses';
+import createTheme from '../styles/createTheme';
+
+const theme = createTheme();
```

Comment:
This is not needed. You can access the `theme` in `styled()` API

Reply:
What do you mean we can access it through the api ? can you give me an example please

Reply:
https://github.com/mui/material-ui/blob/27f1c282cf4a8c5b5aa175b8b594d3a726a8474b/docs/data/material/components/tables/CustomizedTables.tsx#L11